test/specs/views/Items/Item-test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {Item} from 'stardust';
+import faker from 'faker';
+
+describe('Item', () => {
+  it('has children', () => {
```

Comment:
Small semantic request here.  This test isn't testing that `Item has children` but that `Item renders children`.